extension/chrome/settings/modules/backup.ts

```diff
@@ -229,8 +229,18 @@ Catch.try(async () => {
 
   const backupOnEmailProviderAndUpdateUi = async (primaryKi: KeyInfo) => {
     const pp = await Store.passphraseGet(acctEmail, primaryKi.longid);
-    if (!pp || !await isPassPhraseStrongEnough(primaryKi, pp)) {
-      await Ui.modal.warning('Your key is not protected with a strong pass phrase, skipping');
+    if (!parentTabId) {
+      await Ui.modal.error(`Error: There isn't 'parentTabId' property. Please contact human@flowcrypt.com`);
+      return;
+    }
+    if (!pp) {
+      BrowserMsg.send.passphraseDialog(parentTabId, { type: 'backup', longids: [primaryKi.longid] });
+      await Store.waitUntilPassphraseChanged(acctEmail, [primaryKi.longid]);
+      await backupOnEmailProviderAndUpdateUi(primaryKi);
+      return;
+    }
+    if (!isPassPhraseStrongEnough(primaryKi, pp) && await Ui.modal.confirm('Your key is not protected with strong pass phrase, would you like to change pass phrase now?')) {
+      window.location.href = Env.urlCreate('/chrome/settings/modules/change_passphrase.htm', { acctEmail, parentTabId, redirectTo: 'backup' });
```

Comment:
Here it is not clear that change_passphrase handles `redirectTo` parameter at all.

Reply:
Sorry, I forgot to remove it, one second

Reply:
Removed